### PR TITLE
Add more merchant and pirate ship name possibilities

### PR DESCRIPTION
**Pull request by Pointedstick**
I feel like every time I gear up to contribute more content for Endless Sky, I start with the merchant and pirate ship names. Well, here goes!

### CODE REVIEW

File: data/names.txt [hunk 1]
```diff
@@ -963,7 +1017,6 @@ phrase "standalone civilian ship names"
 		"Botany Bay"
 		"Medway"
 		"Allure"
-		"Neptune"
```
Comment (Pointedstick) [inline data/names.txt]: Moved lower down to the section with other planets in it.

File: data/names.txt [hunk 2]
```diff
@@ -1073,6 +1126,124 @@ phrase "standalone civilian ship names"
...
+		"Geofon"
+		"Beowulf"
+		"Nunki"
+		"Mercury"
+		"Venus"
+		"Mars"
+		"Jupiter"
+		"Saturn"
+		"Neptune"
+		"Uranus"
```
Comment (KiLEdEnNis) [inline data/names.txt]: seems you purposefully organized them based on distance from the sun, which means Neptune should be below this one, not above it
(although I remember there was a planet whose trajectory sometimes brings it close to the sun than another planet)

Comment (Pointedstick) [reply data/names.txt]: lol oops

Comment (jafdy) [reply data/names.txt]: Pluto moves inside Neptune's orbit for a short part of the plutonian year.

Comment (Bladewood) [reply data/names.txt]: Sadly, all planetary orbits in Endless Sky are circular.

Comment (MCOfficer) [reply data/names.txt]: Pluto is not a planet, however.
*leans back and grabs popcorn*

File: data/names.txt [hunk 3]
```diff
@@ -1286,6 +1466,141 @@ phrase "civilian"
...
+		""
+		""
+		""
+		""
+		""
+		""
+		""
+		""
+		""
+		"II"
```
Comment (endless-sky) [inline data/names.txt]: I think there should be a space here.

Comment (Pointedstick) [reply data/names.txt]: Oops!

File: data/names.txt [hunk 4]
```diff
@@ -2115,11 +2513,9 @@ phrase "pirate nouns"
 		"Wolf"
 		"Rover"
 		"Rambler"
-		"King"
```
Comment (Pointedstick) [inline data/names.txt]: These all moved to "pirate nouns that work after bad outcomes"

File: data/names.txt [hunk 5]
```diff
@@ -2007,90 +2375,117 @@ phrase "sinister names"
...
+		"Impaler"
+
+phrase "pirate adjectives that don't work as titles"
+	word
+		"Sea"
 		"Morning"
 		"Night"
 		"Evening"
 		"Midnight"
-		"Black"
```
Comment (Pointedstick) [inline data/names.txt]: These all moved to "pirate adjectives that work as titles"

File: data/names.txt [hunk 6]
```diff
@@ -185,6 +185,18 @@ phrase "adjectives"
...
 		"Maori"
 		"Prussian"
+		"Smoky"
+		"Cloudy"
+		"Silly"
+		"Leaky"
+		"African"
+		"Etruscan"
+		"Trojan"
+		"Borneo"
```
Comment (endless-sky) [inline data/names.txt]: "Borneo" isn't an adjective - is this supposed to be "Bornean"?

**Comment by Pointedstick** [comment]
Yay!

**Pull request merged**